PYME/LMVis/views.py

```diff
@@ -153,6 +153,14 @@ def __div__(self, scalar):
                     self.clip_plane_position / scalar
                     )
 
+    def __floordiv__(self, scalar):
+        # Integer division py3
+        return self.__div__(scalar)
+
+    def __truediv__(self, scalar):
+        # Float division py3

```

Comment:
pretty sure you can `//` on py2 and py3, no?

Reply:
Just note that normal int-> float promotion still applies
```
(py27) Andrews-MacBook-Pro-5:~ Andrew$ python
Python 2.7.16 |Anaconda, Inc.| (default, Sep 24 2019, 16:55:38) 
[GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> 2.5 // 2
1.0
>>> 3 // 2
1
>>> quit()
(py27) Andrews-MacBook-Pro-5:~ Andrew$ conda deactivate
Andrews-MacBook-Pro-5:~ Andrew$ conda activate base
(base) Andrews-MacBook-Pro-5:~ Andrew$ python
Python 3.6.9 |Anaconda, Inc.| (default, Jul 30 2019, 13:42:17) 
[GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> 2.5 // 2
1.0
>>> 3 // 2
1
>>> 

```

Reply:
Yes, but `/` is used in the current code. In py2, '/' defaults to `__div__`, in py3 it defaults to `__truediv__`. So, when we do `/` in py3, it throws an error, because it checks `__truediv__`, which is isn't overloaded.